src/v2/Apps/Auctions/Routes/PastAuctions.tsx

```diff
@@ -1,43 +1,146 @@
   return (
-    <Box>
-      {salesConnection.edges.map(({ node }, index) => {
-        return (
-          <Box my={4} key={index}>
-            <AuctionArtworksRailFragmentContainer sale={node} />
-          </Box>
-        )
-      })}
-    </Box>
+    <>
+      <Box>
+        {viewer.salesConnection.edges.map(({ node }, index) => {
+          return (
+            <Box my={4} key={index}>
+              <AuctionArtworksRailFragmentContainer sale={node} />
+            </Box>
+          )
+        })}
+      </Box>
+
+      <Grid my={6}>
+        <Row>
+          <Col sm={6} mx="auto">
+            <Button
+              width="100%"
+              variant="secondaryGray"
+              onClick={handleClick}
+              loading={isLoading}
+              disabled={!relay.hasMore()}
+            >
+              Show more
+            </Button>
+          </Col>
+        </Row>
+      </Grid>
+    </>
   )
 }
 
-export const PastAuctionsFragmentContainer = createFragmentContainer(
+export const PastAuctionsPaginationContainer = createPaginationContainer(
   PastAuctions,
   {
-    salesConnection: graphql`
-      fragment PastAuctions_salesConnection on SaleConnection {
-        edges {
-          node {
-            slug
-            name
-            href
-            endAt
-            ...AuctionArtworksRail_sale
+    viewer: graphql`
+      fragment PastAuctions_viewer on Viewer
+        @argumentDefinitions(
+          first: { type: "Int", defaultValue: 15 }
+          after: { type: "String" }
+        ) {
+        salesConnection(
+          first: $first
+          after: $after
+          published: false
+          live: false
+          sort: START_AT_ASC
+        ) @connection(key: "PastAuctions_salesConnection") {
+          totalCount
+          edges {
+            node {
+              slug
+              name
+              href
+              endAt
+              ...AuctionArtworksRail_sale
+            }
           }
         }
       }
     `,
+  },
+  {
+    direction: "forward",
+    getFragmentVariables(prevVars, totalCount) {
+      return {
+        ...prevVars,
+        count: totalCount,
+      }
+    },
+    getVariables(props, { count, cursor }, fragmentVariables) {
+      return {
+        ...fragmentVariables,
+        count,
+        after: cursor,
+      }
+    },
+    query: graphql`
+      query PastAuctionsQuery($first: Int!, $after: String) {
+        viewer {
+          ...PastAuctions_viewer @arguments(first: $first, after: $after)
+        }
+      }
+    `,
   }
 )
+
+// export const PastAuctionsFragmentContainer = createFragmentContainer(
+//   PastAuctions,
+//   {
+//     salesConnection: graphql`
+//       fragment PastAuctions_salesConnection on SaleConnection {
+//         edges {
+//           node {
+//             slug
+//             name
+//             href
+//             endAt
+//             ...AuctionArtworksRail_sale
+//           }
+//         }
+//       }
+//     `,
+//   }
+// )
```

Comment:
#tip its helpful to click the `files` tab after opening yr PR (or scanning the diff before clicking the "open PR" button) just to give yrself a little self-review; then stuff like this can be caught before needing to be commented on by other devs.

Reply:
Removed!